apiv2/schema/schema.yaml

```diff
@@ -841,6 +841,17 @@ classes:
           - api_s3_path
         range: string
         required: true
+      file_size:
+        name: file_size
+        description: Size of the file in bytes
+        from_schema: cdp-dataset-config
+        exact_mappings:
+          - api_file_size
+        range: integer
+        minimum_value: 0
+        unit:
+          symbol: MB
+          descriptive_name: megabytes
```

Comment:
It's typical to return file sizes in bytes and then convert filesizes to (mb/gb/etc when those sizes are displayed. I'd prefer if we stored sizes in bytes, but if we specifically want to store them in megabytes for some reason, can we change the field names to `file_size_mb` or something to ensure that nobody tries to read them as bytes?